package.json

```diff
@@ -113,7 +113,7 @@
     "preact": "^7.1.0",
     "preact-compat": "^3.9.0",
     "raven-js": "^3.10.0",
-    "react-bosonic": "^1.0.0-beta5",
+    "react-bosonic": "beta",
```

Comment:
truedat. I noticed also that when you just write 'beta', on yarn install, it takes the newer local version and doesn't bother checking on NPM if there's a even newer version…